packages/twenty-server/src/modules/webhook/jobs/call-webhook.job.ts

```diff
@@ -34,10 +47,38 @@ export class CallWebhookJob {
     };
 
     try {
+      const headers: Record<string, string> = {
+        'Content-Type': 'application/json',
+      };
+
+      const {
+        record: { secret, ...recordWithoutSecret } = {},
+        ...dataWithoutRecord
+      } = data;
+
+      const payloadWithoutSecret = {
+        ...dataWithoutRecord,
+        record: recordWithoutSecret,
```

Comment:
You're trying to avoid leaking the secret for webhook.updated/ webhook.created / webhook.deleted events? 
I get your point but I'm not sure it's worse doing what you're doing here. If someone creates a custom field which they name "secret" then they might be frustrated that their webhook is not working correctly. It feels like this could be done at the field type level. But then it would something accessible through API but no webhook... I don't see many other use-cases for that, probably not worth it.
If someone doesn't want those webhook events to be sent they can disable it by explicitely stating which objects they want webhooks to be sent for.